File: TradeMonitor.JavaBackend/src/main/java/com/trademonitor/backend/dto/TradeAnalysisResponseDto.java
package com.trademonitor.backend.dto;

public class TradeAnalysisResponseDto {
    private String tradeId;
    private double riskScore;
    private int latencyMs;
    private String comment;

    public String getTradeId() {
        return tradeId;
    }

    public void setTradeId(String tradeId) {
        this.tradeId = tradeId;
    }

    public double getRiskScore() {
        return riskScore;
    }

    public void setRiskScore(double riskScore) {
        this.riskScore = riskScore;
    }

    public int getLatencyMs() {
        return latencyMs;
    }

    public void setLatencyMs(int latencyMs) {
        this.latencyMs = latencyMs;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }
}

File: TradeMonitor.Core/Dtos/TradeAnalysisResponseDto.cs
namespace TradeMonitor.Core.Dtos
{
    public class TradeAnalysisResponseDto
    {
        public string TradeId { get; set; } = string.Empty;
        public decimal RiskScore { get; set; }
        public int LatencyMs { get; set; }
        public string Comment { get; set; } = string.Empty;
    }
}